Description: Improve option -p documentation.
 Improve the wording of the documentation concerning use of the
 -p,--portability option.
Author: Jonathan Nieder
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=511456
 
--- a/doc/time.texi
+++ b/doc/time.texi
@@ -147,14 +147,18 @@ Use @var{format} as the format string.
 
 @item -p
 @itemx --portability
-Use the following format string, for conformance with POSIX standard
-1003.2:
+Use the following format string:
 
 @example
 real %e
 user %U
 sys %S
 @end example
+ 
+The default output format of time differs widely between
+implementations.  This option (in its short form -p) is supported by all
+POSIX-compliant 'time' implementations to retrieve basic information in
+the described format.
 
 @item -v
 @itemx --verbose
